src/utils/particle/updater.ts

```diff
@@ -166,19 +183,21 @@ export class Updater {
                 particle.radius -= (particle.size.velocity || 0);
             }
 
-            if (particle.radius < 0)
+            if (particle.radius < 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)